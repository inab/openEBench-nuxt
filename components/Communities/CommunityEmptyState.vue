<template>
	<div>
		<v-img :src="illustration" contain max-height="300" />
		<h2 class="mt-10 text-h4 text--secondary text-center">
			No benchmarking results yet...
		</h2>
		<h3 class="mt-5 text-h5 text--secondary text-center">
			Come back soon to see some benchmarking results.
		</h3>
	</div>
</template>

<script>
export default {
	props: {},
	data() {
		return {
			illustration: require('~/static/images/illustrations/empty-state.svg'),
		};
	},
};
</script>
