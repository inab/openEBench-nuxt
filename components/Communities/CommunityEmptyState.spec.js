import { mount } from '@vue/test-utils';
import CommunityEmptyState from './CommunityEmptyState.vue';

const factory = () => {
	return mount(CommunityEmptyState, {
		...createComponentMocks({}),
		mocks: {},
	});
};

describe('CommunityEmptyState', () => {
	it('is instantiated', () => {
		const wrapper = factory();
		expect(wrapper).toBeTruthy();
	});

	it('should match snapshot', () => {
		const wrapper = factory();
		expect(wrapper.html()).toMatchSnapshot();
	});
});
