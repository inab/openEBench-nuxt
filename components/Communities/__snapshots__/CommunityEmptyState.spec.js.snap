// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommunityEmptyState should match snapshot 1`] = `
"<div>
  <div class=\\"v-image v-responsive theme--light\\" style=\\"max-height: 300px;\\">
    <div class=\\"v-responsive__content\\"></div>
  </div>
  <h2 class=\\"mt-10 text-h4 text--secondary text-center\\">
    No benchmarking results yet...
  </h2>
  <h3 class=\\"mt-5 text-h5 text--secondary text-center\\">
    Come back soon to see some benchmarking results.
  </h3>
</div>"
`;
